plugins/modules/external_account.py

```diff
@@ -0,0 +1,381 @@
+# Copyright 2024 Cloudera, Inc. All Rights Reserved.
```

Comment:
2025

Reply:
Also, need shebang and UTF8 - https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#python-shebang-utf-8-coding